rpcs3/util/sysinfo.cpp

```diff
@@ -395,59 +395,69 @@ std::string utils::get_system_info()
 	return result;
 }
 
-std::string utils::get_firmware_version()
+std::string utils::get_firmware_version(bool cache)
```

Comment:
This makes no sense at all.
Just use fxo to cache it in sys_game

Reply:
Done 🫡